master/getting-started/kubernetes/installation/calico.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Installing Calico for policy and networking (recommended)
-canonical_url: 'https://docs.projectcalico.org/v3.7/getting-started/kubernetes/installation/calico'
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/installation/calico'
```

Comment:
I think the canonical URL should be `v3.9` - I don't think it should ever point at master. 

It's meant to point at the latest released version of the doc.